src/core/libraries/fiber/fiber.cpp

```diff
@@ -304,6 +342,42 @@ s32 PS4_SYSV_ABI sceFiberSwitch(OrbisFiber* fiber, u64 arg_on_run_to, u64* arg_o
         return ORBIS_FIBER_ERROR_PERMISSION;
     }
 
+    /* Caller wants to attach context and switch. */
+    if (addr_context != nullptr || size_context != 0) {
```

Comment:
Is it possible to move this to a separate function to avoid code duplication in `sceFiberRunImpl` and `sceFiberSwitchImpl`?